lib/storage/http.js

```diff
@@ -15,27 +15,14 @@
 const http = require('@adobe/httptransfer');
 const { AssetComputeLogUtils, GenericError, RenditionTooLarge } = require('@adobe/asset-compute-commons');
 const { actionName } = require('../action');
-const mime = require('mime-types');
 
-const DEFAULT_CONTENT_TYPE = 'application/octet-stream';
-
-
-async function download(source, file) {
+async function download(asset, file) {
     try {
-        const contentType = mime.contentType(source.name) || DEFAULT_CONTENT_TYPE;
-        console.log(`downloading source: ${AssetComputeLogUtils.redactUrl(source.url)} into ${file}`);
-
-        const headers = {
-            'content-type': contentType
-        };
-        if(source.headers){
-            console.log('using additional custom headers');
-            Object.assign(headers, source.headers);
-        }
+        console.log(`downloading source: ${AssetComputeLogUtils.redactUrl(asset.url)} into ${file}`);
```

Comment:
Rename `downloading source:` to `downloading asset:` as well.

And could log headers here as well, on a new line. Logging undefined if not set would be ok:

```suggestion
        console.log(`downloading asset ${AssetComputeLogUtils.redactUrl(asset.url)} into ${file}\nheaders:`, asset.headers);
```